frontend/scss/organisms/_o-custom-tours-builder.scss

```diff
@@ -1,6 +1,16 @@
 // ------ Custom tours
 
 .custom-tours {
+  // This is to counteract some of the odd behavior with the a17 container
+  // There's obviously a reason for things being the way they are but I feel like htnl and body
+  // should ideally not be doing anything with width: 100vw or centering themselves in the screen using css
+  // It's not accounting for the scroll bar in a way that makes sense to me and seems to cause
+  // horizontal cut-off of content approx half the width of the scroll bar on each side
+  // This approach is probably fine with sufficient padding
+  // We could consider having a special class than normalizes the body/html styles for custom tours
```

Comment:
I'll take a look to see if there's a way other areas of the site handle this. We have other pages that are full-width as well.